fastestimator/util/util.py

```diff
@@ -104,16 +104,13 @@
 
 def parse_string_to_python(val: str) -> Any:
     """Convert a string into a python object.
-
```

Comment:
proposed formatting change doesn't match our other docs

Reply:
I'll rectify this.